### PR TITLE
Refactor image conversions

### PR DESCRIPTION
**Pull request by arnaud-morvan**
* Use convert shell command directly with options from config (resize, extent, quality, etc).
* Remove optimization post-prosessing.

Relate #5 #10

### CODE REVIEW

File: c2corg_images/__init__.py [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
 
+
 # See http://docs.wand-py.org/en/0.4.1/guide/resizecrop.html
 # Max 800, keep aspect ratio
 # Max 120x120 don't keep ratio
 THUMBNAIL_CONFIGS = [
-    {'suffix': 'BI', 'convert': ['-resize', '800x800>']},
-    {'suffix': 'MI', 'convert': ['-resize', '250x250>']},
+    {'suffix': 'BI', 'convert': ['-resize', '800x800>',
+                                 '-quality', '80']},
```
Comment (asaunier) [inline c2corg_images/__init__.py]: Is 80 really enough to get a satisfaying image quality?

In v5 the resized images quality is poor whereas using ``convert`` with quality ranging from 75 to 85 it seems:
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/lib/Images.class.php#L159
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/plugins/sfThumbnailPlugin/lib/sfThumbnail.class.php#L43
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/lib/Images.class.php#L47-L48

What about using quality = 90 for instance?

Comment (arnaud-morvan) [inline c2corg_images/__init__.py]: For an example : 
80 => 182ko
90 => 262ko with visible quality improvement.

The default value is 92, from the ImageMagic docs. The default value in most softwares is 90.

IMHO, we should use 90.

**Comment by arnaud-morvan** [comment]
Quality set to 90, waiting for precise image sizes, but this could be done in another PR.

**Comment by asaunier** [comment]
I have asked the association about the images sizes but I don't think I'll be able to give you an answer soon.
Feel free to merge the PR.

**Pull request merged**